### PR TITLE
Added basic auth options in the M365 AC to disable-basic-authentication-in-exchange-online.md

### PR DESCRIPTION
**Pull request by dariomws**
Closes https://github.com/MicrosoftDocs/OfficeDocs-Exchange/issues/2198.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeOnline/clients-and-mobile-in-exchange-online/disable-basic-authentication-in-exchange-online.md [hunk 1]
```diff
@@ -374,6 +374,11 @@ Date: Wed, 31 Jan 2018 05:15:08 GMT
 Content-Length: 0
 ```
 
+## Manage Basic authentication in the Microsoft 365 Admin Center
+
+In the Microsoft 365 Admin Center, under **Settings** > **Org Settings** > **Modern Authentication** you can designate the protocols in your tenant that no longer require Basic Authentication to be enabled.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/clients-and-mobile-in-exchange-online/disable-basic-authentication-in-exchange-online.md:379]:
```suggestion
In the Microsoft 365 Admin Center, under **Settings** > **Org Settings** > **Modern Authentication**, you can designate the protocols in your tenant that no longer require Basic Authentication to be enabled.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edit.

**Comment by scanum** [comment]
@msdmaguire PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**